proto/cosmos/group/v1/module/v1/module.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package cosmos.group.v1.module.v1;
+
+import "cosmos/app/v1alpha1/module.proto";
+
+// Module is the config object of the group module.
+message Module {
+  option (cosmos.app.v1alpha1.module) = {
+    go_import: "github.com/cosmos/cosmos-sdk/x/group"
+  };
+
+  // max_execution_period defines the max duration after a proposal's voting period ends that members can send a MsgExec to execute the proposal.
+  string max_execution_period = 1;
```

Comment:
I think we should just use google.protobuf.Duration. The yaml will stay the same as it is now